pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -0,0 +1,79 @@
+import json
+import sys
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import transaction
+
+from account import TGAS
+from common.base import Transaction
+
+
+class TransferFT(Transaction):
+
+    def __init__(self, ft, sender, recipient_id, how_much=1, tgas=300):
+        super().__init__()
+        self.ft = ft
+        self.sender = sender
+        self.recipient_id = recipient_id
+        self.how_much = how_much
+        self.tgas = tgas
+
+    def sign_and_deser(self, block_hash):
+        (ft, sender, recipient_id) = self.ft, self.sender, self.recipient_id
+        args = {
+            "receiver_id": recipient_id,
+            "amount": str(int(self.how_much)),
+        }
+        tx = transaction.sign_function_call_tx(
```

Comment:
Nit here and in other places: No need to define a variable `tx`, you can just return the result of `sign_function_call_tx` in the same line.

I think a separate variable can help with readability when it's unclear what the result is, but it's not the case here and it just adds more code.

Reply:
Yeah, I completely agree :+1: